gradle/libs.versions.toml

```diff
@@ -19,14 +20,14 @@ mockito-kotlin = { group = "org.mockito.kotlin", name = "mockito-kotlin", versio
 strikt-core = { group = "io.strikt", name = "strikt-core", version.ref = "strikt" }
 strikt-jackson = { group = "io.strikt", name = "strikt-jackson", version.ref = "strikt" }
 
-openapi-annotation-processor = { group = "io.javalin.community.openapi", name = "openapi-annotation-processor", version.ref = "javalin" }
-javalin-openapi-plugin = { group = "io.javalin.community.openapi", name = "javalin-openapi-plugin", version.ref = "javalin" }
-javalin-redoc-plugin = { group = "io.javalin.community.openapi", name = "javalin-redoc-plugin", version.ref = "javalin" }
+openapi-annotation-processor = { group = "io.javalin.community.openapi", name = "openapi-annotation-processor", version.ref = "openapi" }
+javalin-openapi-plugin = { group = "io.javalin.community.openapi", name = "javalin-openapi-plugin", version.ref = "openapi" }
+javalin-redoc-plugin = { group = "io.javalin.community.openapi", name = "javalin-redoc-plugin", version.ref = "openapi" }
 javalin-testtools = { group = "io.javalin", name = "javalin-testtools", version.ref = "javalin" }
 javalin = { group = "io.javalin", name = "javalin", version.ref = "javalin" }
 javalin-micrometer = { group = "io.javalin", name = "javalin-micrometer", version.ref = "javalin" }
 
-# vulnerabilities in version 11.0.13. Can be removed after updating Javalin and Kotlin
+# vulnerabilities in version 11.0.20. Can be removed after updating Javalin and Kotlin
```

Comment:
note: I think this can be removed along with an explicit bom declaration. The new Javalin version should be without Jetty vulnerabilites